### DISCUSSION TITLE
请问如何在命令行静默执行识别

### DISCUSSION THREAD

**Discussion by zxr201** [hiroi-sora/Umi-OCR#700] [Q&A]
我有个任务就是在业务流程里某一步识别一些pdf信息。测试在调用命令行的时候如果用下面命令是可以读出识别信息的并保存的，但是会出现窗口
umi-ocr --path "E:\ocrTest\Umi-OCR_Paddle_v2.1.4\a.png" --output "a.txt"
如果我用
umi-ocr --hide --path "E:\ocrTest\Umi-OCR_Paddle_v2.1.4\a.png" --output "a.txt"
命令行指UI打印一行 Umi-OCR hide. 并且输出到a.txt里
请问如何可以后台静默识别不弹出窗口。尝试了写bat里面前后调用
umi-ocr --hide
umi-ocr --path "E:\ocrTest\Umi-OCR_Paddle_v2.1.4\a.png" --output "a.txt"
umi-ocr --quit
还是会闪现一下窗口

**Comment by hiroi-sora** [comment]
截图OCR标签页 → 设置 → 识图后的操作 → 弹出主窗口（取消勾选）

![image](https://github.com/user-attachments/assets/5e344f28-67f0-4d87-a7af-e7a7a57a2e79)

**Comment by zxr201** [comment]
成功了，谢谢！
